docs/build/guides/transactions/create-account.mdx

```diff
@@ -625,6 +624,102 @@ public final class main {
 }
 ```
 
+```rust
+use std::{cell::RefCell, rc::Rc, time::Duration};
+
+use soroban_client::{
+    account::{Account, AccountBehavior},
+    asset::{Asset, AssetBehavior},
+    keypair::{Keypair, KeypairBehavior},
+    network::{NetworkPassphrase, Networks},
+    operation::{self, Operation},
+    transaction::{TransactionBehavior, TransactionBuilder, TransactionBuilderBehavior},
+    xdr, Options, Server,
+};
+
+const TESTNET_USDC_ISSUER: &str = "GBBD47IF6LWK7P7MDEVSCWR7DPUWV3NY3DTQEVFL4NAT4AQH3ZLLFLA5";
+
+#[tokio::main]
+pub async fn main() -> Result<(), Box<dyn std::error::Error>> {
+    let server_url = "https://soroban-testnet.stellar.org";
+    let server = Server::new(server_url, Options::default()).expect("Cannot create server");
+
+    //
+    // Generate keypairs
+    //
+    let parent = Keypair::random()?;
+    println!("Secret: {}", parent.secret_key()?);
+    println!("Public: {}", parent.public_key());
+
+    //
+    // Fund a new account via friendbot
+    //
+    let account_data = server.request_airdrop(&parent.public_key()).await?;
+    println!("SUCCESS! You have a new account:\n{account_data:?}");
+
+    let child = Keypair::random()?;
+    let mut parent_account =
+        Account::new(&parent.public_key(), &account_data.sequence_number()).unwrap();
+
+    //
+    // Fund a new account via CreateAccount operation
+    //
+    let mut builder = TransactionBuilder::new(&mut parent_account, Networks::testnet(), None);
+    builder.fee(1000u32);
+    builder.add_operation(
+        Operation::new()
+            .create_account(&child.public_key(), operation::ONE * 5)
+            .unwrap(),
+    );
+
+    let mut tx = builder.build();
+    tx.sign(&[parent]);
+
+    let response = server.send_transaction(tx).await?;
+    let hash = response.hash;
+    println!("Tx hash: {}", hash);
+
+    //
+    // Polling for transaction completion
+    //
+    if server
+        .wait_transaction(&hash, Duration::from_secs(15))
+        .await
+        .is_err()
+    {
+        return Err("Failed to create account".into());
+    }
```

Comment:
You can choose to ignore this nit and I'm not sure if this is formatter correctly.
```suggestion
    server
      .wait_transaction(&hash, Duration::from_secs(15))
      .await
      .map_err(|_| Err("Failed to create account"))?;
```